ui/components/component-library/avatar-account/avatar-account.stories.js

```diff
@@ -25,53 +20,54 @@ export default {
   argTypes: {
     size: {
       control: 'select',
-      options: Object.values(AVATAR_ACCOUNT_SIZES),
+      options: Object.values(AvatarAccountSize).map(
+        (value) => value.toLowerCase(), // Removes reverse mapping from enum this is a temporary fix until we are using typescript for everything
```

Comment:
Without this we will get reverse mapping options in the controls

<img width="912" alt="Screenshot 2023-03-08 at 1 50 26 PM" src="https://user-images.githubusercontent.com/8112138/223858597-e0c06cfc-0133-4a12-8bcc-4128e71157bf.png">